values-pt-rBR/strings.xml

```diff
@@ -1,78 +1,78 @@
 <resources tools:ignore="ExtraTranslation" xmlns:tools="http://schemas.android.com/tools">
-    <string name="translators">"@abacate123"</string> <!--put/add yourself here as the AUTHOR(S) of this translation-->
+    <string name="translators">"@caiostob"</string> <!--put/add yourself here as the AUTHOR(S) of this translation-->
```

Comment:
I hope he apologizes, but his translation was awful. I did not hesitate to take his credits.